dis_snek/models/discord/channel.py

```diff
@@ -845,6 +846,61 @@ def permissions_for(self, instance: Snowflake_Type) -> Permissions:
 
             return self.permissions_for(instance)
 
+    async def add_permission(
+        self,
+        target: Union["PermissionOverwrite", "models.Role", "models.User"],
+        type: "OverwriteTypes",
+        allow: Optional[List["Permissions"]] = None,
+        deny: Optional[List["Permissions"]] = None,
+        reason: Optional[str] = None,
+    ) -> None:
+        """
+        Add a permission to this channel.
+
+        Args:
+            target: The permission target
+            type: The type of permission overwrite
+            allow: List of permissions to allow
+            deny: List of permissions to deny
+            reason: The reason for this change
+
+        Raises:
+            ValueError: Invalid target for permission
+        """
+        allow = allow or []
+        deny = deny or []
+        if not isinstance(target, PermissionOverwrite):
+            target_type = None
+            if isinstance(target, models.User):
```

Comment:
```suggestion
            if isinstance(target, (models.User, models.Member)):
```

`Member` class should apply also

Reply:
Updated